src/parsers/manifestjson.js

```diff
@@ -624,6 +624,18 @@ export default class ManifestJSONParser extends JSONParser {
       }
     }
 
+    if (typeof this.parsedJSON.developer === 'object') {
+      const { name, url } = this.parsedJSON.developer;
+
+      if (name) {
+        this.parsedJSON.author = name;
+      }
```

Comment:
This isn't strictly needed because I don't think we do anything with the `author` yet but it is there to mimic FF's behavior.